I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/loadLabels.rpy", line 28, in script
    $ inventory.selectItem(0)
  File "game/loadLabels.rpy", line 28, in <module>
    $ inventory.selectItem(0)
  File "game/classes.rpy", line 45, in selectItem
    self.slots[index].setSelected(True)
IndexError: list index out of range

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/loadLabels.rpy", line 28, in script
    $ inventory.selectItem(0)
  File "C:\Users\Yann\Downloads\renpy-8.0.3-sdk\renpy\ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Yann\Downloads\renpy-8.0.3-sdk\renpy\python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/loadLabels.rpy", line 28, in <module>
    $ inventory.selectItem(0)
  File "game/classes.rpy", line 45, in selectItem
    self.slots[index].setSelected(True)
  File "C:\Users\Yann\Downloads\renpy-8.0.3-sdk\renpy\revertable.py", line 217, in __getitem__
    rv = list.__getitem__(self, index)
IndexError: list index out of range

Windows-10-10.0.19045 AMD64
Ren'Py 8.0.3.22090809
J 1.0
Thu Feb  9 23:07:20 2023
